### ISSUE TITLE
Cookie not created automatically

### ISSUE THREAD

**Issue by RalpinMcDjango** [lexik/LexikJWTAuthenticationBundle#760]
Hi guys,

As i understand since [#753](https://github.com/lexik/LexikJWTAuthenticationBundle/pull/753) the bundle should automatically return a secure and httpOnly cookie with the name provided in the config YAML.

This doesn't appear to be working on my environment right now. I'm testing this via Postman using Symfony 5 as the framework.

lexik_jwt_authenctication.yaml

# token extraction settings
    token_extractors:
        cookie:
            enabled: true
            name: Bearer
        authorization_header:
            enabled: false
            prefix:  Bearer
            name:    Authorization
        query_parameter:
            enabled: false
            name:    bearer

Any ideas why this might not be working as expected?

**Comment by chalasr** [comment]
Hey,

For backward compatibility reasons, we can't enable the feature based on the configured extractor (since one could have a similar cookie-adding logic which could break when upgrading). 

For enabling this feature, you need to set another config option:
```yaml
token_extractors: 
    cookie: 
        enabled: true
        name: BEARER
# ...
set_cookies:
    BEARER: ~

# Full config with defaults:
#  BEARER:
#      lifetime: null (defaults to token ttl)
#      samesite: lax
#      path: /
#      domain: null (null means automatically set by symfony)
```

**Comment by chalasr** [comment]
If you don't mind, I'll keep this issue open as a reminder, we need to document the new option (PR welcome :) )

**Comment by RalpinMcDjango** [comment]
> Hey,
> 
> For backward compatibility reasons, we can't enable the feature based on the configured extractor (since one could have a similar cookie-adding logic which could break when upgrading).
> 
> For enabling this feature, you need to set another config option:
> 
> ```yaml
> token_extractors: 
>     cookie: 
>         enabled: true
>         name: BEARER
> # ...
> set_cookies:
>     BEARER: ~
> 
> # Full config with defaults:
> #  BEARER:
> #      lifetime: null (defaults to token ttl)
> #      samesite: lax
> #      path: /
> #      domain: null (null means automatically set by symfony)
> ```

That's perfect, thank you. I thought there might be some more configuration required.



> If you don't mind, I'll keep this issue open as a reminder, we need to document the new option (PR welcome :) )

Not a problem at all :)

**Comment by RalpinMcDjango** [comment]
PR submitted

**Comment by chalasr** [comment]
Fixed by #763, Thank you very much @RalpinMcDjango
